lustrefs-exporter/src/brw_stats.rs

```diff
@@ -149,6 +149,30 @@ static BLOCK_MAPS_MSEC_TOTAL: Metric = Metric {
     r#type: MetricType::Counter,
 };
 
+static RECOVERY_STATUS: Metric = Metric {
+    name: "recovery_status",
+    help: "Gives the recovery status on a target.",
```

Comment:
This needs a description between enum value (as int) and what it corresponds to.